lib/x11regressiontest.pm

```diff
@@ -481,9 +481,22 @@ sub exit_firefox() {
 }
 
 sub start_gnome_settings {
-    send_key "super";
-    wait_still_screen;
-    type_string "settings";
+    my $is_sle_sp1 = check_var('VERSION', '12-SP1');
+    my $workaround_repetitions = 3;
+
+    # the loop is a workaround for SP1: bug in launcher. Sometimes it doesn't react to click
+    do {
+        if ($is_sle_sp1) {
+            send_key "super";    # if launcher is open, close it (search string will also be removed).
+            send_key "esc";      # close launcher, if it still open
```

Comment:
In general I recommend to use single ticks when you don't want variable expansion